### ISSUE TITLE
🐛 [firebase_analytics_web] Should be updated to work with firebase 8.0.0

### ISSUE THREAD

**Issue by eggnstone** [FirebaseExtended/flutterfire#5033]
`firebase` is currently at version 8.0.0 which does not work with `firebase_analytics_web` 0.1.1.
Please update `firebase_analytics_web` and `firebase_analytics`.

```
Because firebase_analytics_web 0.1.1 depends on firebase ^7.3.0 and no versions of firebase_analytics_web match >0.1.1 <0.2.0, firebase_analytics_web ^0.1.1 requires firebase ^7.3.0.

So, because xyz depends on both firebase ^8.0.0 and firebase_analytics_web ^0.1.1, version solving failed.
```

**Comment by eggnstone** [comment]
Looks fixed for firebase_analytics_web ^0.2.0

**Comment by markusaksli-nc** [comment]
This was fixed to resolve the recent dependency issues
